spyder/dependencies.py

```diff
@@ -290,9 +299,12 @@ def __init__(self, modname, package_name, features, required_version,
 
     def check(self):
         """Check if dependency is installed"""
-        return programs.is_module_installed(self.modname,
-                                            self.required_version,
-                                            self.installed_version)
+        if self.required_version and self.installed_version:

```

Comment:
Why did you add this line?

Reply:
We do not have a "required version" for spyder plugins, so those deps do not define these values so I skip this check otherwise it would fails with a invalid version string comparison.